cpp/data/installer/ixtheo.cronjobs

```diff
@@ -37,3 +37,4 @@ LOG_DIR=/usr/local/var/log/tuefind
 45 23 * * * "$BIN/rss_subset_aggregator" "--mode=email" "$EMAIL" "relbib" > "$LOG_DIR/rss_subset_aggregator_relbib.log" 2>&1
 {IF DEFINED(production)}0 17 1 Jul,Dec * "$BIN/ch.sh"{ENDIF}
 {IF DEFINED(production)}0 0 1 * * "$BIN/cologne.sh"{ENDIF}
+{IF DEFINED(production)}30 0 * * * "$BIN/generate_new_journal_alert_stats.sh"{ENDIF}
```

Comment:
Similar to other stats programs like collect_solr_stats_data and others, it might be good to run this in non-production mode as well (in case we would like to change something)

Reply:
Done.